spec/support/shared_examples/with_metadata_spec.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+RSpec.shared_examples 'with metadata' do
+  let(:item_count) { 20 }
+  let(:meta_keys) { %w[total limit offset relationships] }
+
+  it 'contains total, limit, offset and relationships keys' do
```

Comment:
Yes, it contains, but what abouts its content(s)? You are only testing the `item_count` below...but the rest is untested :disappointed: